specification/operationalinsights/data-plane/readme.nodejs.md

```diff
@@ -0,0 +1,15 @@
+
+## NodeJS
+These settings apply only when `--nodejs` is specified on the command line.
+Please also specify `--node-sdks-folder=<path to the root directory of your azure-sdk-for-node clone>`.
+
+``` yaml $(nodejs)
+nodejs:
+  package-name: azure-loganalytics
+  package-version: 0.1.0
+  output-folder: $(node-sdks-folder)/lib/services/loganalytics
+  override-client-name: LogAnalyticsClient
+  generate-license-txt: false
+  generate-package-json: false
+  generate-readme-md: false
```

Comment:
@alexeldeib - Are you doing anything custom in the README.md? Trying to understand why generate-* directives are marked false.
@daschult - Should the generate-* be false?

Reply:
See also https://github.com/Azure/azure-sdk-for-node/pull/3008 -- I generated using these directives as "true", but I modified the readme examples by hand. I didn't want these overwritten with the generated readme. Does that sound correct, or have I misunderstood the auto-generation here?

I guess that doesn't apply to package-json or license-txt, I can mark these true again.